src/object.rs

```diff
@@ -571,6 +588,19 @@ impl Object {
   }
 }
 
+/// Object integrity levels can be used to restrict what can be done to an
+/// object's properties.
+#[repr(C)]
+pub enum IntegrityLevel {
```

Comment:
:question: Should this be:

```suggestion
#[derive(Debug)]
#[repr(C)]
pub enum IntegrityLevel {
```

Reply:
Yes :-)